lib/providers/view_model/wallet_detail/fee_bumping_view_model.dart

```diff
@@ -344,75 +384,118 @@ class FeeBumpingViewModel extends ChangeNotifier {
   }
 
   // 추천 수수료
-  int _getRecommendedFeeRate() {
+  double _getRecommendedFeeRate() {
     if (_type == FeeBumpingType.cpfp) {
       return _getRecommendedFeeRateForCpfp();
     }
     return _getRecommendedFeeRateForRbf();
   }
 
-  int _getRecommendedFeeRateForCpfp() {
+  double _getRecommendedFeeRateForCpfp() {
     final recommendedFeeRate = _feeInfos[1].satsPerVb; // 보통 수수료
 
     if (recommendedFeeRate == null) {
       return 0;
     }
 
-    if (recommendedFeeRate <= _transaction.feeRate) {
-      return _transaction.feeRate.toInt() + 1;
+    _generateCpfpTransaction(recommendedFeeRate.toDouble());
+
+    double cpfpTxSize = _estimateVirtualByte(_bumpingTransaction!);
+    double totalFee = (_transaction.vSize + cpfpTxSize) * recommendedFeeRate;
+    double cpfpTxFee = totalFee - _transaction.fee!.toDouble();
+    double cpfpTxFeeRate = cpfpTxFee / cpfpTxSize;
+
+    if (recommendedFeeRate < _transaction.feeRate + 0.1) {
+      return _transaction.feeRate + 0.1;
+    }
+    if (cpfpTxFeeRate < recommendedFeeRate || cpfpTxFeeRate < 0) {
+      return recommendedFeeRate.toDouble();
     }
 
-    return recommendedFeeRate;
+    return cpfpTxFeeRate;
   }
 
-  int _getRecommendedFeeRateForRbf() {
+  double _getRecommendedFeeRateForRbf() {
     final recommendedFeeRate = _feeInfos[2].satsPerVb; // 느린 수수료
 
     if (recommendedFeeRate == null) {
       return 0;
     }
 
-    if (_transaction.feeRate + 1 > recommendedFeeRate) {
-      return _transaction.feeRate.toInt() + 1;
+    double temporalFeeRate = _transaction.feeRate + 1;
+
+    _generateRbfTransaction(temporalFeeRate); // 임시 최소 수수료율로 트랜잭션 생성
+
+    double estimatedVirtualByte = _estimateVirtualByte(_bumpingTransaction!);
+    double expectedFee = _transaction.fee!.toDouble() + estimatedVirtualByte;
+
```

Comment:
double expectedFee = estimatedVirtualByte * _feeInfos[1].satsPerVb; // 보통 수수료;
```suggestion
    double expectedFee = _transaction.fee!.toDouble() + estimatedVirtualByte;

```

Reply:
(rbf에서 기대하는 fee) = (이전 트랜잭션 보다 1 sat/vbyte 높은 fee) = (이전 트랜잭션 fee) + (vsize * 추천 수수료율)
의 뜻으로 의도하고 계산한거예요. 추천 수수료가 계속 1 s/vb여서 누락했네요;;;
```
double expectedFee = _transaction.fee!.toDouble() + estimatedVirtualByte * _feeInfos[1].satsPerVb; 
```
이렇게 수정하면 될 것 같은데 어떤가요?!

Reply:
제 RBF경험 상 1씩 높이기보다 계속 혼잡한 상황이 지속돼서 멤풀 추천 수수료까지 올렸던 경험들이었어요. (수십 단위의 feeRate을 올렸던 것 같습니다.) 또 사람들이 질문했던 상황을 돌이켜보면 멤풀이 지속적으로 혼란스러운데 FeeRate을 1~2 처럼 엄청 낮게 설정한 경우가 많았어요. 이 경우에는 오히려 네트워크의 추천 수수료만큼 RBF를 추천하는 것이 좋겠다고 생각했어요. 

최적의 수수료를 위해서 멤풀 블록 1번째~2번째 아슬아슬한 경우나 멤풀에 여유가 있을 때는 1씩 올리는 게 유효할 수 있겠지만 이것 역시 네트워크 수수료가 맞게 제공될 것으로 생각해요.

Reply:
위 코드는 최소 수수료로 보는게 맞겠네요. 변수명도 바꾸고 메인넷 상황에 맞게 다시 작성해볼게요.